flexmeasures/ui/templates/crud/asset.html

```diff
@@ -221,6 +221,13 @@ <h3>Edit {{ asset.name }}</h3>
                         }
 
                         $(window).ready(() => {
+                            // Set initial date range from session storage
```

Comment:
This here (and the same in sensors.html is not needed right? It's done [here](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures/ui/templates/base.html#L475) and I successfully tested without.

Reply:
Oh i didn't know that, ill tet on my end also, cause before I added this the date range want persisting in the calendar UI

Reply:
Feel free to test without it as I did to confirm

Reply:
So I tested without it and the range wasn't persistent on the date picker UI(it works on the graph but the color isn't showing on the date picker), but I will move this logic to base.html